modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/load/IgniteCapitalizationBenchmark.java

```diff
@@ -0,0 +1,277 @@
+
+    static final String PERSON_CACHE = "person";
+    static final String DEPOSIT_CACHE = "deposit";
+    static SqlFieldsQuery findDepositQuery;
+
+    static {
+        findDepositQuery = new SqlFieldsQuery("SELECT _key FROM \"" + DEPOSIT_CACHE + "\".Deposit WHERE affKey=?").setLocal(true);
+    }
+
+    /** {@inheritDoc} */
+    @Override public void setUp(BenchmarkConfiguration cfg) throws Exception {
+        super.setUp(cfg);
+
+        preLoading();
+    }
+
+    /**
+     * @throws Exception If fail.
+     */
+    private void preLoading() throws Exception {
+
+        Thread preloadPerson = new Thread() {
+            @Override public void run() {
+                setName("preloadPerson");
+
+                try (IgniteDataStreamer dataLdr = ignite().dataStreamer(PERSON_CACHE)) {
+                    for (int i = 0; i < args.preloadAmount() && !isInterrupted(); i++)
+                        dataLdr.addData(i, createPerson(i));
+                }
+            }
+        };
+
+        preloadPerson.start();
+
+        Thread preloadDeposit = new Thread() {
+            @Override public void run() {
+                setName("preloadDeposit");
+
+                try (IgniteDataStreamer dataLdr = ignite().dataStreamer(DEPOSIT_CACHE)) {
+                    for (int i = 0; i < args.preloadAmount() && !isInterrupted(); i++) {
+                        int personId = nextRandom(args.preloadAmount());
+
+                        dataLdr.addData(new AffinityKey(i, personId), createDeposit(i, personId));
+                    }
+                }
+            }
+        };
+
+        preloadDeposit.start();
+
+        preloadDeposit.join();
+
+        preloadPerson.join();
+    }
+
+    /**
+     * @param id Identifier.
+     * @return Person entity as binary object.
+     */
+    private BinaryObject createPerson(int id) {
+        BinaryObjectBuilder clientBuilder = ignite().binary().builder("Person");
+
+        clientBuilder.setField("firstName", "First name " + id);
+
+        clientBuilder.setField("lastName", "Last name " + id);
+
+        clientBuilder.setField("age", id);
+
+        clientBuilder.setField("personId", id);
+
+        return clientBuilder.build();
+    }
+
+    /**
+     * @param id Identifier.
+     * @return Deposit entity as binary object.
+     */
+    private BinaryObject createDeposit(int id, int affKey) {
+        BinaryObjectBuilder clientBuilder = ignite().binary().builder("Deposit");
+
+        clientBuilder.setField("accountNumber", String.format("%s%s-%s%s",id,id,id,id));
+
+        clientBuilder.setField("amount", (long)(id % 1000));
+
+        clientBuilder.setField("depositId", id);
+
+        clientBuilder.setField("affKey", affKey);
+
+        return clientBuilder.build();
+    }
+
+
+
+    /** {@inheritDoc} */
+    @Override public boolean test(Map<Object, Object> ctx) throws Exception {
+
+        Map<UUID, List<Integer>> partitionsMap = personCachePartitions();
```

Comment:
This can be done once at setUp state. We suppose that the topology won't change during the test.